### PR TITLE
Add CVE-2021-43803 for GHSA-25mp-g6fv-mqxx

### PR DESCRIPTION
**Pull request by shelbyc**
Add CVE-2021-43803 for GHSA-25mp-g6fv-mqxx

### CODE REVIEW

File: 2021/43xxx/CVE-2021-43803.json [hunk 1]
```diff
@@ -1,18 +1,111 @@
 {
```
Comment (cve-team) [inline 2021/43xxx/CVE-2021-43803.json:1]: Duplicate Refs Detected(same URL)

**Review by cve-team — CHANGES_REQUESTED** [review]
Auto validation errors:

**Comment by ktodd229** [comment]
Please verify that the requested changes have been made and resubmitted.

File: 2021/43xxx/CVE-2021-43803.json (hunk 1 above)
Comment (shelbyc) [reply 2021/43xxx/CVE-2021-43803.json:1]: Duplicate URL removed!

**Pull request merged**